var/spack/repos/builtin/packages/py-cylc-uiserver/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCylcUiserver(PythonPackage):
+    """Cylc UI Server."""
+
+    homepage = "https://github.com/cylc/cylc-uiserver/"
+    pypi = "cylc-uiserver/cylc-uiserver-1.3.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("1.3.0", sha256="f3526e470c7ac2b61bf69e9b8d17fc7a513392219d28baed9b1166dcc7033d7a")
+
+    depends_on("python@3.7:3.9", type=("build", "run"))
+    depends_on("py-wheel", type="build")
```

Comment:
`py-wheel` is automatically added by the `PythonPackage` class and you only need to add it again as dependency if a specific version is needed.